plugin.go

```diff
@@ -81,6 +88,56 @@ func (p Plugin) Exec() error {
 	return nil
 }
 
+func customCertHandler(certPath string) *exec.Cmd {
+	if IsUrl(certPath) {
+		if downloadCert(certPath) {
+			return setCustomCert("/tmp/customCert.pem")
```

Comment:
just add a const for the string `/tmp/customCert.pem` and use it :)

else it's good to go